tests/core/test_nlg.py

```diff
@@ -115,6 +115,19 @@ def test_nlg_fill_template_text(slot_name, slot_value):
     assert result == {"text": str(slot_value)}
 
 
+@pytest.mark.parametrize(
+    "img_slot_name, img_slot_value",
+    [("url", "https://www.exampleimg.com"), ("img1", "https://www.appleimg.com")],
+)
+def test_nlg_fill_template_image(img_slot_name, img_slot_value):
```

Comment:
can we also test the case where we have a `text` and and `image` present and both templates have to be filled?

Reply:
Done. Please have a look at new commits